.github/workflows/checks.yml

```diff
@@ -345,31 +345,6 @@ jobs:
           npx mocha test.js
           cd ../../../../..
 
-  check_no_std:
```

Comment:
Are we sure about removing this?

I am guessing we keep this to make sure the guest code compiles without native features even if in some sov-sdk crates we don't need the check anymore.